filament/backend/src/metal/MetalDriver.mm

```diff
@@ -416,12 +416,28 @@
 
 void MetalDriver::createIndexBufferR(Handle<HwIndexBuffer> ibh, ElementType elementType,
         uint32_t indexCount, BufferUsage usage) {
+    // Errors are postponed until the next tick, to give Filament a chance to call setDebugTag on
+    // the handle; this way we get nicer error messages.
+
+    // Check that the elementType is valid.
+    const bool isSupportedElementType =
+            elementType == ElementType::USHORT || elementType == ElementType::UINT;
+    if (UTILS_UNLIKELY(!isSupportedElementType)) {
+        runAtNextTick([this, ibh, isSupportedElementType, elementType]() {
```

Comment:
nit: `isSupportedElementType` feels redundant since we're sure it's false at this point.

Reply:
Good catch, so my thinking here was to make the Panic's literal more helpful. Otherwise is just has "false" which isn't very descriptive.

Reply:
We should catch these errors on the filament side. There are very few places where we  call createIndexBuffer (probably only one). If done on the filament side, it should never ever happen to be wrong on the backend side.